zebra-chain/src/block/header.rs

```diff
@@ -81,11 +81,36 @@ impl BlockHeader {
 
         Ok(())
     }
+
+    /// Check if `self.time` is less than or equal to
+    /// 2 hours in the future, according to the node's local clock (`now`).
+    ///
+    /// This is a non-deterministic rule, as clocks vary over time, and
+    /// between different nodes.
+    ///
+    /// "In addition, a full validator MUST NOT accept blocks with nTime
+    /// more than two hours in the future according to its clock. This
+    /// is not strictly a consensus rule because it is nondeterministic,
+    /// and clock time varies between nodes. Also note that a block that
+    /// is rejected by this rule at a given point in time may later be
+    /// accepted."[S 7.5][7.5]
```

Comment:
```suggestion
    /// accepted." [§7.5][7.5]
```

Reply:
Note to self: set up Unicode fonts in my emacs instance.